### PR TITLE
add publish

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "qless-js",
+  "name": "qless-js-inst",
   "version": "0.4.10",
```
Comment (migmit) [inline package.json:3]: Um... why is `version` in `package-lock.json` different?

Comment (vitali-makarevich) [reply package.json:3]: fixed, thank you

**Review by migmit — APPROVED** [review]

**Comment by migmit** [comment]
Guess I don't have write access for my review to count.

**Review by lucheol — APPROVED** [review]
LGTM

**Pull request merged**